SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -359,9 +359,20 @@ bool FSpatialGDKEditorToolbarModule::ValidateGeneratedLaunchConfig() const
 		return false;
 	}
 
-	int32 PlayNumberOfServers;
-	GetDefault<ULevelEditorPlaySettings>()->GetPlayNumberOfServers(PlayNumberOfServers);
-	if (!SpatialGDKRuntimeSettings->bEnableHandover && PlayNumberOfServers > 1)
+	const ULevelEditorPlaySettings* LevelEditorPlaySettings = GetDefault<ULevelEditorPlaySettings>();
+
+	bool bLaunchingMultipleServerWorkers = false;
+	if(const int32* ServerCount = LevelEditorPlaySettings->WorkerTypesToLaunch.Find(SpatialConstants::ServerWorkerType))
+	{
+		if (*ServerCount > 1)
+		{
+			bLaunchingMultipleServerWorkers = true;
+		}
+	}
+
+	bLaunchingMultipleServerWorkers = bLaunchingMultipleServerWorkers || LevelEditorPlaySettings->WorkerTypesToLaunch.Num() > 1;
+	
+	if (!SpatialGDKRuntimeSettings->bEnableHandover && bLaunchingMultipleServerWorkers)
```

Comment:
I'm not sure if this check is valid anymore since pure offloading doesn't need handover. Do you know if handover is required for offloading to worker? If not we could modify this to check if there is >1 of any single worker type.